src/cmd/group.go

```diff
@@ -38,9 +39,161 @@ EOF
 	},
 }
 
-// TODO: Get Group
+var getGroupCommand = &cobra.Command{
+	Use:        "group ID|ALIAS",
+	Short:      "Get details about a group",
+	Long:       `Get details about a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		common.PrettyPrint(group)
+		common.WasFound(group.Id == nil, key)
```

Comment:
this `WasFound` method needs to come before the `PrettyPrint` so the error is thrown first

Reply:
I thought the same thing when I checked to see what it did. I'll move it to before and check where I got this from to see if the same needs to be done elsewhere.